java/core/src/test/org/apache/orc/TestVectorOrcFile.java

```diff
@@ -4025,7 +4025,7 @@ public void testPredicatePushdownWithNan() throws Exception {
 
     DoubleColumnVector dbcol = ((DoubleColumnVector) batch.cols[0]);
 
-    // first row NaN (resulting to both min and max columnStats of stride to be NaN)
+    // first row NaN (resulting to  min/max and sum columnStats of stride to be NaN)
```

Comment:
`to  min` -> `to min`? (Removing extra space)

Reply:
sure, fixed!